bigbluebutton-config/bin/bbb-conf

```diff
@@ -1747,7 +1747,7 @@ if [ -n "$HOST" ]; then
 
     if [ -f $HTML5_DEFAULT_CONFIG ]; then
         yq w -i $HTML5_DEFAULT_CONFIG public.kurento.wsUrl "wss://$HOST/bbb-webrtc-sfu"
-        yq w -i $HTML5_DEFAULT_CONFIG public.note.url      "$PROTOCOL://$HOST/pad"
+        yq w -i $HTML5_DEFAULT_CONFIG public.pads.url      "$PROTOCOL://$HOST/pad"
```

Comment:
Shouldn't we stop modifying the config file which is shipped by the packages and change the override file `/etc/bigbluebutton/bbb-html5.yml` instead?

Reply:
Agreed.